language-design.md

```diff
@@ -17,7 +12,49 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> "After all, the very existence
+of a master plan means, by definition, that the members of the community
+can have little impact on the future shape of their community, because most
+of the important decisions have already been made. In a sense, under a master
+plan people are living with a frozen future, able to affect only relatively 
+trivial details." [Steele, 1998]
+
+This quote by Christopher Alexander is something I definitely agree with. Before
+reading this article, I did not think about this much and I would have maybe 
+held the opinion that it would be good to have a master plan and have the team
+working on the language determine all aspects of the design. However, I thought
+about the Go community in particular and how they are very transparent and
+take suggestions and implement them in the language regularly. This creates,
+of course, a great community, but also a decent indicator as to what features
+are needed generally versus more "niche features" as mentioned in the Steele
+article.
+
+> “Every piece of software we use was written by other people, slaving away over
+thousands of lines of code. Why shouldn’t those "interfaces" be as humanely
+designed as the ones we tap and swipe?“ [Pavlus, 2012]
+
+While I think this quote makes a valid point, I feel that it oversimplifies the
+situation significantly. When you provide minimalistic controls so as not to
+confuse your user, you sacrifice a lot of control. As an example, we could compare
+Microsoft Word and LaTeX. I don’t think anyone would disagree that Word is more user
+friendly and easier to learn. However, as soon as you want to do something a
+little more technical than what it was designed to handle, it becomes a pain to try
+and configure it to do exactly what you want. So there is an important balance to be
+struck with increasing usability without sacrificing power.
+
```

Comment:
I think you make a good point here about the tradeoff between power and usability, and I do think that LaTeX is pretty well-designed and user friendly for what it is. To play devil's advocate a bit, do you think LaTeX could have been designed better? The fact that Word (user-friendly) and LaTeX (powerful) are all we have does not mean that we could not, in principle, have a language that does both. I think the authors of Quorum would argue that we could, or at least that we should strive for more of a balance.